modules/client-configuration/pages/clients-slemicro.adoc

```diff
@@ -3,27 +3,39 @@
 
 This section contains information about registering clients running {sle-micro} operating systems 5.1, 5.2, 5.3, 5.4, and 5.5 on {x86_64}, {arm64}, and {zsystems} (s390x) architectures.
 
-The {sle-micro} is an ultra-reliable, lightweight operating system purpose built for edge computing.
-It leverages the enterprise hardened security and compliance components of SUSE Linux Enterprise and merges them with a modern, immutable, developer-friendly OS platform.
-
 The {sle-micro} uses transactional updates.
-
 Transactional updates are atomic (all updates are applied only if all updates succeed) and support rollbacks.
 They do not affect a running system as no changes are activated until after the system is rebooted.
-This information is displayed in the menu:Systems[Details > Overview] subtab.
 
 For more information on transactional updates and rebooting, see
 https://documentation.suse.com/sles/html/SLES-all/cha-transactional-updates.html.
 
+// FIXME: We either must add this to the other client files (snippet), or
+//        up to the suse registration page
+The client registration described in this chapter, consists of these steps:
+
+* Synchronize software channels for the new client's operating system and architecture by using either {webui} or command line interface.
+
+ifeval::[{uyuni-content} == true]
+
+* Synchronize the client tool repository for the newly added operating systems by using command line interface.
+
+endif::[]
+
+* Run the actual registration.
+
+* Additional sections to check and verify the registration procedure.
```

Comment:
This feels like it's missing a verb.

Did you want to write maybe something like:

```suggestion
Additional sections describe ways to check and verify the registration procedure.
```